autogen/agentchat/contrib/agent_eval/agent_eval.py

```diff
@@ -0,0 +1,101 @@
+from typing import Dict, List, Optional, Union
+
+import autogen
+from autogen.agentchat.contrib.agent_eval.criterion import Criterion
+from autogen.agentchat.contrib.agent_eval.critic_agent import CriticAgent
+from autogen.agentchat.contrib.agent_eval.quantifier_agent import QuantifierAgent
+from autogen.agentchat.contrib.agent_eval.subcritic_agent import SubCriticAgent
+from autogen.agentchat.contrib.agent_eval.task import Task
+
+
+def generate_criteria(
+    llm_config: Optional[Union[Dict, bool]] = None,
+    task: Task = None,
```

Comment:
is it required to be a `Task` type or can it be optional?

Reply:
going to leave it as required to be a task for now, we might revisit it later